build_debian.sh

```diff
@@ -79,6 +79,8 @@ TARGET_PATH=$TARGET_PATH scripts/build_debian_base_system.sh $CONFIGURED_ARCH $I
 # Prepare buildinfo
 sudo scripts/prepare_debian_image_buildinfo.sh $CONFIGURED_ARCH $IMAGE_DISTRO $FILESYSTEM_ROOT $http_proxy
 
+sudo chown root $FILESYSTEM_ROOT
```

Comment:
Why suddenly need to chown? Do you want to change the group at the same time?

Reply:
Yes, also changed the group to root